### PR TITLE
setup.py: Use our version of roguehostapd, not PyPI’s

### PR DESCRIPTION
**Pull request by cclauss**

### CODE REVIEW

**Comment by sophron** [comment]
I should have merged this PR first. I'll resolve the conflicts later today.

File: .travis.yml [hunk 1]
```diff
@@ -4,16 +4,13 @@ python:
...
 
-matrix:
-  allow_failures:
-    - python: "3.7"
-
 install:
-  - pip install pylint tox scapy
-  - pip install tornado==4.5.3
+  - pip install flake8 pylint scapy tornado tox
+  - pip install git+http://github.com/wifiphisher/roguehostapd.git
```
Comment (sophron) [inline .travis.yml]: Interesting, I wasn't aware this is possible. Me and @anakin1028 spent a lot of time troubleshooting with PyPi.

**Comment by cclauss** [comment]
> This branch has no conflicts with the base branch

**Comment by cclauss** [comment]
Sync with upstream

**Comment by sophron** [comment]
Thank you @cclauss.

**Pull request merged**